src/Domain/Creators/TimeDependence/Shape.hpp

```diff
@@ -0,0 +1,184 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <limits>
+#include <memory>
+#include <optional>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/TimeDependent/Shape.hpp"
+#include "Domain/Creators/TimeDependence/GenerateCoordinateMap.hpp"
+#include "Domain/Creators/TimeDependence/TimeDependence.hpp"
+#include "Options/Auto.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/ErrorHandling/Assert.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace domain::FunctionsOfTime {
+class FunctionOfTime;
+}  // namespace domain::FunctionsOfTime
+namespace domain::CoordinateMaps::TimeDependent {
+class Shape;
+}  // namespace domain::CoordinateMaps::TimeDependent
+namespace domain {
+template <typename SourceFrame, typename TargetFrame, typename... Maps>
+class CoordinateMap;
+}  // namespace domain
+
+namespace Frame {
+struct Distorted;
+struct Grid;
+struct Inertial;
+}  // namespace Frame
+/// \endcond
+
+namespace domain::creators::time_dependence {
+/*!
+ * \brief A Shape conforming to a Kerr horizon, as given by
```

Comment:
please update these docs so they reflect that the inner surface conforms to a surface of constant Boyer-Lindquist radius `r_BL` in Kerr-Schild coordinates. The value of `r_BL` is `inner_radius`.